cli/src/test/kotlin/io/spine/protodata/cli/app/CliTest.kt

```diff
@@ -107,9 +115,10 @@ class `Command line application should` {
             "-p", TestPlugin::class.jvmName,
             "-r", UnderscorePrefixRenderer::class.jvmName,
             "--src", srcRoot.toString(),
+            "--target", targetRoot.toString(),
             "-t", codegenRequestFile.toString()
```

Comment:
In scope of future PRs, let's deal with this `-t`. I have read the description for this parameter (a `--request` in its full form). But still, it's a bit confusing, since `-t` does not correspond to anything except for "requesT".

Could you please file an issue on this matter?

Reply:
I've filed an [issue](https://github.com/SpineEventEngine/ProtoData/issues/162).